src/Tasks.UnitTests/XslTransformation_Tests.cs

```diff
@@ -68,12 +68,10 @@ public sealed class XslTransformation_Tests
         private readonly string _xslDocument = "<xsl:stylesheet version=\"1.0\" xmlns:xsl=\"http://www.w3.org/1999/XSL/Transform\" xmlns:msxsl=\"urn:schemas-microsoft-com:xslt\" exclude-result-prefixes=\"msxsl\"><xsl:output method=\"xml\" indent=\"yes\"/><xsl:template match=\"@* | node()\"><surround><xsl:copy><xsl:apply-templates select=\"@* | node()\"/></xsl:copy></surround></xsl:template></xsl:stylesheet>";
 
 
-#if FEATURE_COMPILED_XSL
```

Comment:
The `FEATURE_COMPILED_XSL` is defined allways, except for Mono: https://github.com/dotnet/msbuild/blob/cc3db358d34ad4cd1ec0c67e17582d7ca2a15040/src/Directory.BeforeCommon.targets#L24
Since we no longer support Mono (per @ladipro), we can remove all occurences of this symbol alltogether

Reply:
These constants are defined in net472, not including net7 since condition `<PropertyGroup Condition="$(TargetFramework.StartsWith('net4')) Or $(TargetFramework.StartsWith('net3'))">`  we also need to verify they work on .net7 before removing all occurrences of this symbol altogether
https://github.com/dotnet/msbuild/blob/6300d22b25cc1bcb634663bd7087e31d8312bb15/src/Directory.BeforeCommon.targets#L17-L74

Reply:
Sorry my bad, you are right. Making sure we are supporting precompiled XSLT on core as well is out of scope of this PR - so please disregard this comment